code/modules/client/preference_setup/loadout/loadout_uniform.dm

```diff
@@ -371,30 +371,17 @@
 	display_name = "sexy maid uniform"
 	path = /obj/item/clothing/under/dress/sexymaid
 
-/datum/gear/uniform/pt
-	display_name = "pt uniform"
-	path = /obj/item/clothing/under/pt
-
-/datum/gear/uniform/pt/expeditionary
-	display_name = "pt uniform, sifguard"
-	path = /obj/item/clothing/under/pt/expeditionary
-
-/datum/gear/uniform/pt/fleet
-	display_name = "pt uniform, fleet"
-	path = /obj/item/clothing/under/pt/fleet
-
 /datum/gear/uniform/utility
 	display_name = "utility, black"
 	path = /obj/item/clothing/under/utility
-	cost = 2
 
-/datum/gear/uniform/utility/fleet
-	display_name = "utility, navy"
```

Comment:
I assumed 'navy' was referring to it in the same way as the 'navy' security suit- though, the clarification here can't hurt.

Reply:
though clarifying the 'navy' security suit to 'blue' would also maybe be a good idea

Reply:
I removed the term "Navy" here because it might be confused with the actual Navy, which is sort of what the Fleet is.

Reply:
fair